api/initialize.py

```diff
@@ -1,3 +1,55 @@
-from app.models import db
+from app.models import db, User, Election, Question
+from app.config import TEST_SERVER, MEMBERSHIP_API, SEED_DATA, TOKEN
+import json
+import requests
+import datetime
 
-db.create_all()
\ No newline at end of file
+db.drop_all()
+db.create_all()
+
+if(SEED_DATA):
+
+    User.query.delete()
+    db.session.commit()
+    print(TOKEN)
```

Comment:
Continuing on the topic of the TOKEN variable from the config file, I think it's best to store the JWT in the configuration file as a string, and apply it in the relevant dictionary here:
```suggestion
    r = requests.get(MEMBERSHIP_API + 'api/v1/leaderboard', headers = {"Authorization": f"Bearer {TOKEN}"})
```